Lib/test/test_regrtest.py

```diff
@@ -1326,7 +1326,7 @@ def test_crashed(self):
                                   parallel=True, stats=0)
 
     def parse_methods(self, output):
-        regex = re.compile("^(test[^ ]+).*ok$", flags=re.MULTILINE)
+        regex = re.compile("^test_regrtest_noop[\d]+\.Tests\.(test[^ .]+).*ok$", flags=re.MULTILINE)
```

Comment:
This likely needs an `r"` prefix on the string for the `\`s to do what you intended.  there's probably a SyntaxWarning.

Also, in the modified regexes in this file, why use `[\d]+` rather than `\d+`?

Reply:
True, fixed.